content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -154,37 +154,38 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ```
 
-You cannot modify which `Role` or `ClusterRole` a binding object refers to.
-Attempts to change the `roleRef` field of a binding object will result in a validation error.
-To change the `roleRef` field on an existing binding object, the binding object must be deleted and recreated.
+After you create a binding, you cannot change the Role or ClusterRole that it refers to.
+If you try to change its `roleRef` you will get a validation error. Instead, you would need
+to remove the binding object and create a replacement.
+
 There are two primary reasons for this restriction:
 
+1. Making `roleRef` immutable allows granting someone `update` permission on an existing binding
+object, so that they can manage the list of subjects, without being able to change
+the level of access that is granted to those subjects.
```

Comment:
previous text was more accurate... it keeps them from changing the role that is granted